gz-msgs-extras.cmake.in

```diff
@@ -0,0 +1,78 @@
+# Copyright 2023 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# copied from gz-msgs/gz-msgs-extras.cmake
+
+find_package(Python3 REQUIRED COMPONENTS Interpreter)
+
+include(${@PROJECT_NAME@_DIR}/gz_msgs_string_utils.cmake)
+include(${@PROJECT_NAME@_DIR}/gz_msgs_protoc.cmake)
+include(${@PROJECT_NAME@_DIR}/gz_msgs_factory.cmake)
+include(${@PROJECT_NAME@_DIR}/gz_msgs_generate.cmake)
+include(${@PROJECT_NAME@_DIR}/target_link_messages.cmake)
+
+set(@PROJECT_NAME@_INSTALL_PATH "${@PROJECT_NAME@_DIR}/../../..")
```

Comment:
given the other issues I've been seeing with the `/usr/lib/<arch>` folder in https://github.com/gazebo-release/gz-msgs10-release/pull/3 and https://github.com/gazebosim/gz-cmake/pull/360, I'm a bit suspicious that this hard-coded `../../..` is wrong